stubs/PyYAML/yaml/__init__.pyi

```diff
@@ -49,7 +51,7 @@ def emit(
 @overload
 def serialize_all(
     nodes,
-    stream: IO[str],
+    stream: _WriteStream[Any],
```

Comment:
Shouldn't these be `_WriteStream[str]`?

Reply:
As far as I can see, the type of dependent on `encoding` and the existence of `stream.encoding`. One possible future improvement is to add an overload for when `encoding` is `str`.